### ISSUE TITLE
Still alive?

### ISSUE THREAD

**Issue by palicepta** [Pryx/server-status#165]
Is this project still alive?

I tried to install it with docker, and I have a problem with adding New Service or New Service group.
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/112872766/192774292-a2fbc7b6-9e53-4133-a995-e320eb037c32.png">

Will be great if it is still alive, and anybody helping me with my pain :)

**Comment by KorenMarvin** [comment]
Hi Yes,
i am still developing on it. I also have it a more modern look.
![Schermafbeelding 2022-10-13 082657](https://user-images.githubusercontent.com/36482784/195518555-cb2a4a50-1173-43d9-bfa7-4e40b6be2e26.png)

**Comment by KorenMarvin** [comment]
Last one ;)
![image](https://user-images.githubusercontent.com/36482784/195566670-71288ffa-c134-41f3-a9e9-a42aca108591.png)

**Comment by palicepta** [comment]
Hello,
It looks very cool, just keep it up.
Thank you very much :)

**Comment by brenofebronio001** [comment]
Hello, will you make this layout available?

**Comment by KorenMarvin** [comment]
Yes i'll need to minify and structure it a bit and then i will publish it.

**Comment by palicepta** [comment]
Its good news, very much looking forward.

**Comment by brenofebronio001** [comment]
Very good! Do you have a release date?

**Comment by cs2000** [comment]
Hey, just wondering if this project is still being developed? The screenshots of the new interface looked really good and i am excited to see where the project goes. but not seen any activity for a long time so just wanted to check!

**Comment by Steffen-MLR** [comment]
> Yes i'll need to minify and structure it a bit and then i will publish it.

Hi, i can merge this, if you create a pull request :)
